plugins/in_winevtlog/in_winevtlog.c

```diff
@@ -261,6 +281,11 @@ static struct flb_config_map config_map[] = {
       0, FLB_TRUE, offsetof(struct winevtlog_config, event_query),
       "Specify XML query for filtering events"
     },
+    {
+      FLB_CONFIG_MAP_STR, "read_limit_per_cycle", NULL,
```

Comment:
This should be a FLB_CONFIG_MAP_SIZE instead of STR, with a proper default

Reply:
Oh, it should be nice.

Reply:
I fixed it in https://githu.b.com/fluent/fluent-bit/pull/8677/commits/540faf226e5f1688209585a99010230d9103b05b